src/algopy_testing/_context_helpers/ledger_context.py

```diff
@@ -114,21 +114,21 @@ def update_asset(self, asset_id: int, **asset_fields: typing.Unpack[AssetFields]
             raise ValueError("Asset not found in testing context!")
         self.asset_data[asset_id].update(asset_fields)
 
-    def get_app(self, app: algopy.UInt64 | int) -> algopy.Application:
+    def get_app(self, app_id: algopy.UInt64 | int) -> algopy.Application:
```

Comment:
I think it might be worth removing the `get_app_for_contract` on the main context and just making this method accept an `algopy.Contract`. That way there is just one method for getting an existing app.